apps/website-new/docs/en/_meta.json

```diff
@@ -14,15 +14,15 @@
     "link": "/plugin/dev/index",
     "activeMatch": "/plugin/"
   },
+  {
+    "text": "Blog",
+    "link": "/blog/showcase",
+    "activeMatch": "/blog/"
+  },
   {
     "text": "Community",
-    "link": "/community/showcase",
     "activeMatch": "/community/",
     "items": [
-      {
```

Comment:
why are we removing showcase?

Reply:
showcase was added above

Reply:
In the blog, I am still thinking about how to put the document here will be better